quoridor-app/app/api/gameApi.js

```diff
@@ -30,3 +30,24 @@ export const joinGame = async gameId => {
     }
   });
 };
+
+export const movePawn = async (authToken, gameId, position) => {
```

Comment:
As in other languages you can use functions with multiple arguments when there is 1 or 2 arguments.
But when there is more than 2 arguments it is a good practice to use a single object argument: `{authToken, gameId, position}` in your case